### DISCUSSION TITLE
Hide title

### DISCUSSION THREAD

**Discussion by Laxy317** [toeverything/blocksuite#1947] [General]
how to hide the page title ?

**Comment by doodlewind** [comment]
If you are only using our editor as an inline input box, you can checkout https://github.com/toeverything/blocksuite/blob/master/packages/virgo/README.md :)

**Reply by Laxy317** [reply]
I need to use `@blocksuite/editor`, and I want to use a custom title that I can add some elements between title and frame.So I want to hide the default page title

**Reply by himself65** [reply]
You can try to overwrite the CSS to hide it. We don't have plan to remove the title component for now

**Reply by Laxy317** [reply]
ok, thanks.
